kumascript/macros/APIRef.ejs

```diff
@@ -159,37 +155,45 @@ var badges = {
   ObsoleteBadge : await template("ObsoleteBadge"),
 }
 
+function getPageTitle(page) {
+  if (page.shortTitle) {
+    return page.shortTitle;
+  }
+  else {
+    const titleSplit = htmlEscape(page.title).split('.'); // Two cases, as sometimes the interface name is forgotten in the title:
+    let title = titleSplit[titleSplit.length - 1];       // "WebGLRenderingContext.activeTexture()" and "activeTexture()" should both become "activeTexture()"
+    // Event pages have a title like "Interface: eventname event" which looks silly
+    // in the sidebar. So for event pages we use the slug, which is supposed to be
+    // in the form "eventname_event", and split off "_event" to leave us (hopefully)
+    // with just "eventname" for the link text.
+    if (page.pageType === 'web-api-event') {
+      title = page.slug.split('/').pop();
+      if (title.endsWith('_event')) {
+          title = title.slice(0,-6);
+      }
+    }
+    return title;
+  }
```

Comment:
...this is all temporary, I guess, as we will have `short-title` set for all Web/API pages.